### PR TITLE
added peakfinder

### PR DESCRIPTION
**Pull request by smarkesini**
Added peakfinder in the push_SFX.py script. 
The pull_SFX.py will save the peaks and if test_flag = True will check against the hdf5 output. 
These scripts are called with push_SFX.sh pull_SFX.sh

The pull_SFX.py requires a "sfx_raw_1_2_3" detector class in the psana test_detectors.py:
https://github.com/slac-lcls/lcls2/blob/master/psana/psana/detector/test_detectors.py

### CODE REVIEW

**Comment by chuckie82** [comment]
@monarin I can help with code review if your prefer, but it seems @smarkesini and I need to be added to the list of "Collaborators".

**Comment by monarin** [comment]
Hi,

I added both of you as collaborators. Let me know if you see any problems.

Mona

File: xtc1to2/read_img.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def __init__(self, exp, run, mode, detector_name):
 
         # Set up detector
         self.detector = psana.Detector(detector_name)
+        # set flag (for Chuck)
+        self.detector.do_reshape_2d_to_3d(flag=True)
```
Comment (monarin) [inline xtc1to2/read_img.py:49]: I guess I don't understand this but I think @chuckie82 and @smarkesini know where this function needs to be added.

**Review by monarin — APPROVED** [review]
Everything looks okay to me. For the new detector class in test_detectors.py, you can create a pull request and I can run pytest on it.

**Pull request merged**